__tests__/app.test.js

```diff
@@ -0,0 +1,43 @@
+const app = require('../app.js');
+const request = require('supertest');
+const db = require('../db/connection.js');
+const seed = require('../db/seeds/seed.js');
+const testData = require('../db/data/test-data/index.js')
+
+afterAll(() => db.end());
+
+beforeEach(() => seed(testData));
+
+describe('3. GET /api/topics', () => {
+    test('endpoint responds with status of 200 and an array of topic objects', () => {
```

Comment:
the 3 here relates to the ticket number but I'd remove that reference in the test suite